README.md

```diff
@@ -124,6 +124,30 @@ This is particularly useful if you need to produce multiple aXe reports within
 the same test as it would otherwise keep replacing the same report every time
 you run the assertion.
 
+Two report types are supported: HTML and JSON. You can pass options for each
+separately:
+
+```js
+await expect(page.locator('#my-element')).toBeAccessible({
+  reports: {
+    html: {
+      // By default, this library attaches the HTML report on failure,
+      // but you can override this behaviour with attach='on' or 'off'
+      attach: 'retain-on-failure', // 'on' | 'off' | 'retain-on-failure'
+      filename: 'axe-report-on-my-element.html',
+    },
+    json: {
+      attach: 'on', // 'on' | 'off' | 'retain-on-failure'
+      filename: 'axe-report-on-my-element.json',
+      space: 0, // third argument to JSON.stringify
```

Comment:
Space doesn't need to be an option.  I'm fine with it always being 2 space indented.